ui/handle_input.go

```diff
@@ -660,3 +678,14 @@ func (win *Window) sortTransactions() {
 		return frontTime.Before(backTime)
 	})
 }
+
+func (win *Window) updateToTransactionDetailsButtons() {
+	if win.walletTransactions.Total > 0 {
+		win.inputs.toTransactionDetails = make(map[string]*gesture.Click, win.walletTransactions.Total)
+		for _, walTxs := range win.walletTransactions.Txs {
+			for _, txn := range walTxs {
+				win.inputs.toTransactionDetails[txn.Txn.Hash] = &gesture.Click{}
+			}
+		}
+	}
+}
```

Comment:
This be restructured to the new page implementation. The page should handle its own events.